docs/developments/reminder-implementation.md

```
@@ -106,7 +106,7 @@ export async function GET() {
 
 ## 3. 環境変数
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)